src/content/release/archive.md

```diff
@@ -2,57 +2,115 @@
 title: Flutter SDK archive
 short-title: Archive
 description: "All current Flutter SDK releases: stable, beta, and main."
-toc: false
+toc: true
 ---
 
-The {{site.sdk.channel | capitalize }} channel contains the
-most stable Flutter builds.
-To learn more, check out [Flutter's channels][].
-
 {% render docs/china-notice.md %}
 
-To learn what's new in the major Flutter releases,
-check out the [release notes][] page.
+## Overview
+
+The Flutter SDK archive is a collection of all previous versions of the
+Flutter SDK. This archive is useful for developers who need to use an older
+version of Flutter for compatibility reasons or to investigate bugs.
+
+The archive includes Flutter SDKs for Windows, macOS, and Linux on the
+following [channels][]:
+
+*   **{{site.sdk.channel | capitalize }} channel**: This channel contains the
+    most stable Flutter builds. Roughly every third beta version is promoted
+    to the stable version. The stable channel is the recommended channel for
+    new users and for production app releases.
+
+*   **Beta channel**: This channel is the most recent version of Flutter that is
+    available, but it is not yet stable. The beta branch is usually released
+    on the first Wednesday of the month. A fix will typically end up in the
+    beta channel about two weeks after it lands in the main channel. 
+    Releases are distributed as [installation bundles][].
+
+*   **Main channel**: This channel has the newest features, but it hasn't been fully
+    tested and might have some bugs. We don't recommend using it unless you're
+    contributing to Flutter itself.
+
+The following information is available for each Flutter release in the
+SDK archive:
+
+*   **Flutter version**: The version number of the Flutter SDK
+    (e.g., 3.0.0, 2.10.5). This follows semantic versioning, indicating the
+    significance of changes between releases.
+*   **Architecture**: The processor architecture the SDK is built for
+    (e.g., x64, arm64). This specifies the type of processor the SDK is
+    compatible with.
+*   **Ref**: The git commit hash that uniquely identifies the specific codebase
+    used for that release.
+*   **Release Date**: The date when that particular Flutter version was
+    officially released.
+*   **Dart version**: The corresponding version of the Dart programming language
+    included in that Flutter SDK release.
+*   **Provenance**: Provides details about the build process and origin of the
+    SDK, potentially including information about security attestations or
+    build systems used. To view this information in a readable format, run the
+    following command. Replace `[provenance URL]` with the provenance URL and
+    make sure you have [jq](https://stedolan.github.io/jq/) installed.
```

Comment:
I'm not sure we need to maintain instructions on how to parse JSON, this looks like it could easily get out of date since we don't maintain `jq`.

Reply:
Removed those sentences and just let people know that the results use the JSON format.